harvester/ogm_repositories_config.yaml

```diff
@@ -0,0 +1,121 @@
+# aardvark
+edu.wisc:
+  name: "University of Wisconsin-Madison"
+  metadata_format: gbl1
+  filename_regex: |
+    .+?/HeldBy_.+?/.+?\.json
+edu.uwm:
+  name: "University of Wisconsin-Milwaukee"
+  metadata_format: aardvark
+  file_directory: "metadata-aardvark"
```

Comment:
Could we theoretically do `file_directory` as regex to streamline the methods?

Reply:
We could -- and considered this -- but I worry it could be prone to typos where a large percentage (most actually) _can_ use the `file_directory` strategy, but a few do _need_ the `filename_regex` option.

Example filename regex, not terribly pretty:
```
filename_regex: |
    .+?/HeldBy_.+?/.+?\.json
```

But most can just do this:
```
file_directory: "metadata-aardvark"
```

Good suggestion, but I'd propose keeping the flexibility at this time.

Reply:
A-ok!